plugins/woocommerce/i18n/states.php

```diff
@@ -174,8 +174,8 @@
 		'BG-19' => __( 'Silistra', 'woocommerce' ),
 		'BG-20' => __( 'Sliven', 'woocommerce' ),
 		'BG-21' => __( 'Smolyan', 'woocommerce' ),
-		'BG-23' => __( 'Sofia', 'woocommerce' ),
-		'BG-22' => __( 'Sofia-Grad', 'woocommerce' ),
+		'BG-23' => __( 'Sofia-District', 'woocommerce' ),
```

Comment:
One small change (to align with [CLDR R41](https://github.com/unicode-org/cldr/blob/release-41/common/subdivisions/en.xml#L490)):

```suggestion
		'BG-23' => __( 'Sofia District', 'woocommerce' ),
```